### DISCUSSION TITLE
opus 251

### DISCUSSION THREAD

**Discussion by ibanez00** [TeamNewPipe/NewPipe#10762] [Questions] [answered]
Is it possible to download audio sources to the maximum extent?  YT audio source opus 251, this app detects opus 160, can it download the real opus audio source bit rate?  Thanks

**Comment by AudricV** [comment, marked as answer]
251 is the YouTube audio stream identifier (itag), not its bitrate. The 160kbps value shown in the app is an average value and not the real bitrate value of the stream, which can be lower or higher depending on videos.

No change on the bitrate or codec from streams delivered by YouTube is made by the app, so the real bitrate of the itag 251 is downloaded and kept.

**Reply by ibanez00** [reply]
So how do you know what format and bit rate yt uses?  So 160 Audio is the best choice?  Thanks

**Reply by opusforlife2** [reply]
Yes, 160 kbps Opus is the best quality audio provided by YT. But for very old videos, you might notice that the m4a is better. That will be very rare, though.
